Source/WebCore/platform/graphics/cg/ImageDecoderCG.h

```diff
@@ -83,6 +83,7 @@ class ImageDecoderCG final : public ImageDecoder {
 #endif
     
     bool m_isAllDataReceived { false };
+    bool m_decodeMaskImage { false };
```

Comment:
Do we need to have this to be `std::atomic<bool>`? This bool is still accessible from multiple threads.

Reply:
Sure, we can do that.